core/interop/IDynamicInterfaceCastable/README.md

```diff
@@ -0,0 +1,75 @@
+---
+languages:
+- csharp
+- cpp
+products:
+- dotnet-core
+page_type: sample
+name: "IDynamicInterfaceCastable: Supporting Interfaces Dynamically"
+urlFragment: "idynamicinterfacecastable"
+description: "A .NET application that shows how to implement IDynamicInterfaceCastable to project a native object as implementing different managed interfaces."
+---
+
+# `IDynamicInterfaceCastable` Sample
+
+The [`IDynamicInterfaceCastable` API](https://docs.microsoft.com/dotnet/api/system.runtime.interopservices.idynamicinterfacecastable) was introduced in .NET 5 as a way for creating a .NET class that supports interfaces which are not in its metadata.
+
+This sample provides an implementation of `IDynamicInterfaceCastable` that projects a native object as implementing different known managed interfaces. It uses COM conventions (e.g. `QueryInterface`) for interacting with the native object, but does not require the actual COM system.
```

Comment:
The guidelines don't prefer e.g.
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg

Reply:
```suggestion
This sample provides an implementation of `IDynamicInterfaceCastable` that projects a native object as implementing different known managed interfaces. It uses COM conventions (such as `QueryInterface`) for interacting with the native object, but does not require the actual COM system.
```